### ISSUE TITLE
cmatrices lib path

### ISSUE THREAD

**Issue by pieper** [Radiomics/pyradiomics#244]
I did a fresh install of [the current version](https://github.com/Radiomics/pyradiomics/commit/0dd46a7bb5f401018f0e6b094afe7926dab026b2) on a linux VM (Amazon Linux AMI release 2016.09) following the installation instructions.

Then importing to python gives this error:

```
In [1]: import radiomics
Error loading C extensions, switching to python calculation:
Traceback (most recent call last):
  File "radiomics/__init__.py", line 233, in <module>
    from radiomics import _cmatrices as cMatrices
ImportError: cannot import name _cmatrices
```

As a workaround, this allows me to import:

`sys.path.append('/home/pieper/pyradiomics/build/lib.linux-x86_64-2.7/radiomics')
`

Should we add this to the documentation or is there a better way to solve this?

**Comment by JoostJM** [comment]
@pieper, what is `radiomics.__path__`? It is possible/likely that this points to your repository (i.e. it's running in development mode). If so, first run `python setup.py develop` to make sure the C extensions are also available in source code.

I thought I included this in the FAQ, but apparently not, I'll correct this in a future PR.

**Comment by pieper** [comment]
> what is radiomics.__path__?

```
In [2]: radiomics.__path__
Out[2]: ['radiomics']
```

Thanks @JoostJM , running `sudo python setup.py develop` resolves the issue.

I created a pull request to update the docs.  Can you confirm the wording is appropriate and merge?

**Comment by JoostJM** [comment]
Resolved by #245 and #233
